admin_solutions/authentication.adoc

```diff
@@ -5,10 +5,303 @@ toc::[]
 {product-title} supports many different authentication methods, as defined in
 xref:../install_config/configuring_authentication.adoc#install-config-configuring-authentication[Configuring Authentication]:
 
+- xref:basic-auth-remote[Basic Authentication (Remote)]
+- xref:request-header-auth[Request Header]
 - xref:keystone-auth[Keystone]
 - xref:ldap-auth[LDAP]
 - xref:github-auth[GitHub]
 
+[[basic-auth-remote]]
+== Basic Authentication (Remote)
+
+[[BasicAuthPasswordIdentityProvider]]
+
+Basic Authentication is a generic backend integration mechanism that allows
+users to log in to {product-title} with credentials validated against a remote
+identity provider.
+
+[CAUTION]
+====
+Basic Authentication must use an HTTPS connection to the remote server in order
+to prevent potential snooping of the user ID and password, and to prevent
+man-in-the-middle attacks.
+====
+
+With `*BasicAuthPasswordIdentityProvider*` configured, users send their username
+and password to {product-title}, which then validates those credentials against
+a remote server by making a server-to-server request, passing the credentials as
+a Basic Auth header. This requires users to send their credentials to
+{product-title} during login. 
+
+[NOTE]
+====
+This only works for username/password login mechanisms, and {product-title} must
+be able to make network requests to the remote authentication server. 
+====
+
+[[configuring-basic-auth-on-master]]
+=== Configuring Authentication on the Master
+
+. If you have:
+.. Already completed the installation of Openshift, then copy the
```

Comment:
These substeps (1a and 1b) are options rather than substeps, correct? Can they be marked up with points rather than steps? It's one or the other, so not sure if numbered list is right?